packages/editor/src/client/blocks/linkPageBreakBlock.tsx

```diff
@@ -13,46 +34,218 @@ export function LinkPageBreakBlock({
   autofocus,
   disabled
 }: LinkPageBreakBlockProps) {
-  const {text, linkText, linkURL} = value
+  const {
+    text,
+    linkText,
+    linkURL,
+    styleOption,
+    layoutOption,
+    richText,
+    linkTarget,
+    hideButton,
+    image,
+    templateOption
+  } = value
   const focusRef = useRef<HTMLTextAreaElement>(null)
+  const focusInputRef = useRef<HTMLInputElement>(null)
+
+  /* eslint-disable i18next/no-literal-string */
+  /* eslint-disable @typescript-eslint/no-non-null-assertion */
 
-  const {t} = useTranslation()
+  // const {t} = useTranslation()
 
   useEffect(() => {
     if (autofocus) focusRef.current?.focus()
   }, [])
 
+  const handleRichTextChange = useCallback(
+    (richText: React.SetStateAction<RichTextBlockValue>) =>
+      onChange(value => ({
+        ...value,
+        richText: isFunctionalUpdate(richText) ? richText(value.richText) : richText
+      })),
+    [onChange]
+  )
+
+  const [isChooseModalOpen, setChooseModalOpen] = useState(false)
+  const [isEditModalOpen, setEditModalOpen] = useState(false)
+
   return (
     <>
-      <Box flexGrow={1}>
-        <TypographicTextArea
-          ref={focusRef}
-          variant="h2"
-          placeholder={t('blocks.linkPageBreak.text')}
-          align="center"
-          value={text}
-          disabled={disabled}
-          onChange={e => onChange({...value, text: e.target.value})}
-        />
-      </Box>
-      <TypographicTextArea
-        ref={focusRef}
-        variant="body1"
-        placeholder={t('blocks.linkPageBreak.linkText')}
-        align="center"
-        value={linkText}
-        disabled={disabled}
-        onChange={e => onChange({...value, linkText: e.target.value})}
-      />
-      <TypographicTextArea
-        ref={focusRef}
-        variant="subtitle2"
-        placeholder={t('blocks.linkPageBreak.linkURL')}
-        align="center"
-        value={linkURL}
-        disabled={disabled}
-        onChange={e => onChange({...value, linkURL: e.target.value})}
-      />
+      <div style={{display: 'flex', flexFlow: 'row wrap'}}>
```

Comment:
My main problem with this block is that we loose the `WYSIWYG` approach for this block. Since you need to touch this part again because of the UI change i suggest that you move all the Selects, booleans and Image selection to a Drawer. You can use the `embedEditPanel.tsx` as an inspiration.

Reply:
ok, but the current panel UX if not really satisfying (more clicks, no close on overlay click, etc) and the will be a UI replace. The current UX is more easy and faster to use for editors so i suggest to just keep it as it is as you ll touch it in the UI replacement process

once the panel actions are more user friendly this can be reranged better.

Reply:
The UI platform works with the same panel concept. For the forseable future we'll stick with that. New UI branch is almost ready. Will be merged this week. That means you can already merge it into your branch and replace the input elements and move them to the panel.

Reply:
Merge this before you chnage the UI, so that you can dont block the reintegration with you UI changes

Reply:
I will merge this once it's ready.